lib/api/element-commands/_waitForDisplayed.js

```diff
@@ -4,8 +4,17 @@ const {BrowserName} = require('../../utils');
 class WaitForDisplayed extends WaitForElement {
   protocolAction() {
     if (this.api.capabilities) {
-      const {browserName} = this.api.capabilities;
-      if (browserName.toLowerCase() === BrowserName.SAFARI) {
+      // FIXME: temporary fix to get safari working
+      const {browserName, version} = this.api.capabilities;
+      let {browserVersion} = this.api.capabilities;
+
+      if (version) {
+        browserVersion = version;
+      }
+      browserVersion = parseInt(browserVersion, 10);
+
+      // Are we running 12 or newer?
+      if (browserName.toLowerCase() === BrowserName.SAFARI && (browserVersion >= 12) && this.settings.webdriver.start_process === true) {
         return this.executeProtocolAction('getElementProperty', ['hidden']).then(result => ({
           value: result.value === false
         }));
```

Comment:
You'll want to update it to include `status`, otherwise it won't work on BrowserStack, I'm pretty sure.
```suggestion
        return this.executeProtocolAction('getElementProperty', ['hidden']).then(result => ({
          value: result.value === false,
          status: result.status
        }));
```